test/openAPI/features/wallet_linked_authenticate.feature

```diff
@@ -45,3 +45,41 @@ Feature: The endpoint to check the correctness of the data.
     And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_no_of_challenges"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of duplicated transactionId
+    Given Wants to check the correctness of the data
+    And The link code is generated before POST /linked-authorization/authenticate
+    When Send POST /linked-authorization/authenticate request with given requestTime, linkedTransactionId, individualId, "PIN" as authFactorType, "password" as challenge, "alpha-numeric" as format
+    And The second authenticate flow for linkedTransactionId ends
+    Then Receive a response for reuse linkedTransactionId from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response for reuse linkedTransactionId should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response for reuse linkedTransactionId should have status 200
+    And The /linked-authorization/authenticate endpoint response for reuse linkedTransactionId should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response for reuse linkedTransactionId should match json schema with errors
+    And The /linked-authorization/authenticate response for reuse linkedTransactionId should contain errorCode property equals to "invalid_transaction_id"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because of invalid requestTime
+    Given Wants to check the correctness of the data
+    And The link code is generated before POST /linked-authorization/authenticate
+    When Send POST /linked-authorization/authenticate request with given invalid requestTime, linkedTransactionId, individualId, "PIN" as authFactorType, "password" as challenge, "alpha-numeric" as format
+    Then Receive a response from the /linked-authorization/authenticate endpoint
+    And The /linked-authorization/authenticate endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/authenticate endpoint response should have status 200
+    And The /linked-authorization/authenticate endpoint response should have content-type: application/json header
+    And The /linked-authorization/authenticate endpoint response should match json schema with errors
+    And The /linked-authorization/authenticate response should contain errorCode property equals to "invalid_request"
+
+  @negative
+  Scenario: Not able to check the correctness of the data because auth failed
+    Given Wants to check the correctness of the data
+    And The link code is generated before POST /linked-authorization/authenticate
+    When Send POST /linked-authorization/authenticate request with given requestTime, linkedTransactionId, individualId, "PIN" as invalidAuthFactorType, "password" as challenge, "alpha-numeric" as format
```

Comment:
this when statement is confusing, if this is for (https://github.com/GovStackWorkingGroup/bb-identity/pull/35#discussion_r1223796963) comment, then we need to give invalid password for the PIN auth factor and expect auth failed